### PR TITLE
Added CSP configs for 2018 URD T5 Aura, T5 Bayro and T5 Maures

### PR DESCRIPTION
**Pull request by of17**
Created and shared by URD with permission to upload (GTP forum: https://www.gtplanet.net/forum/threads/assetto-corsa-pc-mods-general-discussion.307899/page-2669#post-13448256)

### CODE REVIEW

File: config/cars/mods/urd/urd_t5_maures_2018.ini [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
+
+[BOUNCED_INTERIOR_LIGHT]
+MULT=1,1,1,1
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_maures_2018.ini:12]: Why are you adding white bounced light to the interior?

File: config/cars/mods/urd/urd_t5_maures_2018.ini [hunk 2]
```diff
@@ -0,0 +1,92 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_maures_2018.ini:9]: These values are unreasonably high, please reconsider the usage of MULT= above 2.

I'd recommend testing with several mainstream post-process filters to see if the results align with reasonable brightness one would expect from a car such as this.

File: config/cars/mods/urd/urd_t5_bayro_2018.ini [hunk 3]
```diff
@@ -0,0 +1,96 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_bayro_2018.ini:2]: This is included by default, there's no need to redefine it unless there's an error in guessing.

File: config/cars/mods/urd/urd_t5_bayro_2018.ini [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_bayro_2018.ini:9]: Values are unreasonably high, re-test using several different post-process filters such as Sol to ensure that the visuals aren't broken for other users as BLOOM_LUMINANCE and other glare-related parameters may dim or brighten the emissives.

File: config/cars/mods/urd/urd_t5_aura_2018.ini [hunk 5]
```diff
@@ -0,0 +1,97 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_aura_2018.ini:2]: No need to include this unless there's been an error in guessing.

File: config/cars/mods/urd/urd_t5_aura_2018.ini [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```
Comment (RustyChest) [inline config/cars/mods/urd/urd_t5_aura_2018.ini:9]: Emissive multiplier strength seems high, re-check with different filters to ensure visual consistency for rollout.

**Review by RustyChest — CHANGES_REQUESTED** [review]
State why some of the values outlined in the review were used, please.

File: config/cars/mods/urd/urd_t5_maures_2018.ini (hunk 1 above)
Comment (of17) [reply config/cars/mods/urd/urd_t5_maures_2018.ini:12]: I assume the original author (URD) copied these values from one of the examples linked in the CSP wiki.

File: config/cars/mods/urd/urd_t5_bayro_2018.ini (hunk 3 above)
Comment (of17) [reply config/cars/mods/urd/urd_t5_bayro_2018.ini:2]: probably also copied from one of the examples (e.g. ks_maserati_quattroporte.ini)

**Review by RustyChest — APPROVED** [review]
Good enough, if you would like to keep adding CSP features to cars I would look into PBR!

**Pull request merged**